event-adapter/cmd/server/start.go

```diff
@@ -59,15 +62,16 @@ func NewCommandStartSampleAdapterServer(out, errOut io.Writer, stopCh <-chan str
 	o.Authentication.AddFlags(flags)
 	o.Authorization.AddFlags(flags)
 	o.Features.AddFlags(flags)
-
+	fmt.Println(o.RemoteKubeConfigFile)
```

Comment:
Do you intend to leave it here, or is it just for debugging purposes? If the former, please consider adding a message to explain what you're logging. Also note that we usually use glog.Infof() for logging.